app/controllers/content.js

```diff
@@ -22,6 +22,12 @@ export default Ember.Controller.extend({
              this.set('authors', contributors));
     }),
 
+    applyMathJax: Ember.observer('model', function() {
+        Ember.run.next(this, function() {
```

Comment:
Based on our discussion- ew, routes are way less smart than components. That is annoying.

There doesn't appear to be a direct hook for after dom elements are inserted, but there *is* potentially a more appropriate way to use `Ember.run`. Rather than waiting until the next runloop happens to run (and hoping things are set up the way you want), you can very explicitly identify when something should be run:

http://emberjs.com/api/classes/Ember.run.html#method_next
> ...Note that there are often alternatives to using run.next. For instance, if you'd like to schedule an operation to happen after all DOM element operations have completed within the current run loop, you can make use of the afterRender run loop queue...